src/Native/WebGL.js

```diff
@@ -203,7 +205,7 @@ var _elm_community$elm_webgl$Native_WebGL = function () {
     A2(_elm_lang$core$List$map, function (elem) {
       dataFill(array, attributeInfo.size, data_idx, elem, attribute.name);
       data_idx += attributeInfo.size * elem_length;
-    }, bufferElems);
+    }, _elm_lang$core$List$reverse(bufferElems));
```

Comment:
@nacmartin I'm curious about this particular change. 

* Won't it affect the other cases?
* Could we use `foldr` to not iterate over the list twice?